docs/monitoring.md

```diff
@@ -199,6 +199,8 @@ These metrics are exposed by Celeborn worker.
     - CommitFilesTime
         - The time for a worker to flush buffers and close files related to specified shuffle.
     - SlotsAllocated
+    - ActiveSlotsCount
+      - The number of slots being actively in use 
```

Comment:
Align the style with `- The time for a worker to flush buffers and close files related to specified shuffle`.

Reply:
@CodingCat, any update?

Reply:
sorry, missed this one, just updated

Reply:
@CodingCat, add the indent to align the style with `- The time for a worker to flush buffers and close files related to specified shuffle`.

Reply:
updated, could you take another look?